interfaces/builtin/desktop_test.go

```diff
@@ -91,6 +94,40 @@ func (s *DesktopInterfaceSuite) TestAppArmorSpec(c *C) {
 	c.Assert(spec.SecurityTags(), HasLen, 0)
 }
 
+func (s *DesktopInterfaceSuite) TestMountSpec(c *C) {
+	restore := release.MockOnClassic(false)
+	defer restore()
+
+	// On all-snaps systems, no mount entries are added
+	spec := &mount.Specification{}
+	c.Assert(spec.AddConnectedPlug(s.iface, s.plug, nil, s.coreSlot, nil), IsNil)
+	c.Check(spec.MountEntries(), HasLen, 0)
+
+	// On classic systems, a number of font related directories
+	// are bind mounted from the host system if they exist.
+	release.OnClassic = true
```

Comment:
Typically we'd do this in the testsuite instead:
```
// on a classic system with desktop slot coming from the core snap,
// a number of font related directories are bind mounted from the
// host system if they exist.
restore = release.MockOnClassic(true)
defer restore()
```

Reply:
I'm already using `MockOnClassic` earlier to save the state.  I'll switch this case to use MockOnClassic too, just in case someone splits the test later and leaks state.